dao/src/main/java/greencity/entity/HabitFact.java

```diff
@@ -17,9 +17,9 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    @OneToMany(cascade = {CascadeType.ALL}, mappedBy = "habitFact", fetch = FetchType.LAZY)
+    @OneToMany(cascade = {CascadeType.ALL}, mappedBy = "habitFact", fetch = FetchType.EAGER)
     private List<HabitFactTranslation> translations;
 
-    @ManyToOne
+    @ManyToOne(fetch = FetchType.EAGER)
```

Comment:
Use EAGER only when every request in your application needs this filed. In other case use LAZY and if you need this field  when connection to DB is closed use custom query with inner join

Reply:
fixed